crates/turborepo-telemetry/src/events.rs

```diff
@@ -0,0 +1,40 @@
+use serde::{Deserialize, Serialize};
+
+/// All possible telemetry events must be included in this enum
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub enum TelemetryEvent {
+    Fallback(Fallback),
+    KeyVal(KeyVal),
+}
+
+/// Individual events are defined here
+#[derive(Debug, Clone, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
```

Comment:
I believe this macro is a noop since none of the fields in the struct have an `_`.
```suggestion
```